.github/workflows/pytest.yml

```diff
@@ -10,27 +10,38 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.7]
+        python-version: [3.7.9, 3.8.5]
         #os: ["ubuntu-latest", "macos-latest"]
         os: ["ubuntu-latest"]
 
     runs-on: "${{ matrix.os }}"
-
+    
+    name: Python version ${{ matrix.python-version }} 
     steps:
     - uses: actions/checkout@v2
-
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v1
+    - name: Set up Conda
+      uses: s-weigand/setup-conda@v1
       with:
-        python-version: ${{ matrix.python-version }}
+         activate-conda: true 
+    - run: conda --version
+    - run: conda install python=${{matrix.python-version}}
 
-    - name: Install dependencies
+    - name: Install dependencies with conda
       run: |
-        python -m pip install --upgrade pip
-        pip install -r requirements.txt
-        pip install .
-        pip install scipy
-        echo "::add-path::${GITHUB_WORKSPACE}/bin"
+         conda install --file requirements.txt 
```

Comment:
I know this was done before, but I think the correct way to go is to use conda install .  (in the ipi directory)
And this will install the dependencies automatically

Reply:
I install  I-PI as the following:
conda install -f requirements.txt
conda install pip
pip install .

Reply:
ok